examples/evm/its-interchain-token/index.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
```

Comment:
I'm confused what this demonstrates that isn't in the other folders? this one doesn't have any custom solidity code.

Reply:
removed